doc/logs_doc.doxy

```diff
@@ -270,6 +270,21 @@
  *   - {TYPE}   To display here the type of the log.
  *   - {DATE}   To display here the date of the log.
  *   - {THREAD} To display here the emiter thread number (only if -pthread or -fopenmp).
+ * 
+ * @fn void mlog::Options::bindThreadName(const std::thread::id& id, const std::string& name)
+ * Binds the thread id to the name so that when using the tag
+ * {THREAD} the name given is used instead of an hexadecimal output
+ *
+ * @param[in]  id    The identifier returned by std::thread::get_id()
+ * @param[in]  name  Name that should be displayed instead of an id
+ * @see mlog::Options::unbidThreadName
+ * 
+ * @fn void mlog::Options::unbidThreadName(const std::thread::id& id)
```

Comment:
A n is missing right here, (unbid instead of unbind), please add it in order to get consistent documentation.